test/ViewExtensionLibraryTests/resources/libraryItems.json

```diff
@@ -5664,38 +5664,6 @@
       "itemType": "action",
       "keywords": ""
     },
-    {
-      "fullyQualifiedName": "List.RemoveKey",
```

Comment:
can you check if this file actually comes from the librariejs repo? please keep in sync if you can.

Reply:
This one doesn't seem like it is from the lib repo but I will check

Reply:
This one isn't in the Library Repo